app/src/models/status.ts

```diff
@@ -301,3 +379,10 @@ function getIncludeAllState(
 
   return includeAll
 }
+
+/** Typechecker helper to identify conflicted files */
+export function isConflictedFile(
```

Comment:
This would live in `app/src/lib/status.ts` but I opened #6242 which also creates this file so I plan to rebase this on top of that merged PR and then move it in there (yay merge conflicts!).